### PR TITLE
added human readable time in HH:MM:SS.ms

### PR DESCRIPTION
**Pull request by Ayushsunny**
improved timing printing in python in HH:MM:SS.ms
issue code - #284

### CODE REVIEW

File: Python/tigre/algorithms/art_family_algorithms.py [hunk 1]
```diff
@@ -98,10 +98,16 @@ def run_main_iter(self):
...
                     tic = default_timer()
+
+                    # Esitmated time until completetion in HH:MM:SS.ms
+                    remaining_time = ((self.niter - 1) * (tic - toc))
+                    seconds = int(remaining_time)
+                    milliseconds = str(remaining_time - seconds)[2:5]
                     print(
-                        "Esitmated time until completetion (s): "
-                        + str((self.niter - 1) * (tic - toc))
+                        "Esitmated time until completetion : "
```
Comment (AnderBiguri) [inline Python/tigre/algorithms/art_family_algorithms.py:107]: Typo, as explained in the other PR.  Same in all the others.

**Review by AnderBiguri — CHANGES_REQUESTED** [review]

File: Python/tigre/algorithms/art_family_algorithms.py (hunk 1 above)
Comment (Ayushsunny) [reply Python/tigre/algorithms/art_family_algorithms.py:107]: `**"Esitmated time until completetion in HH:MM:SS.ms : "**` like this sir ?

Comment (AnderBiguri) [reply Python/tigre/algorithms/art_family_algorithms.py:107]: No.... check the words in English, they are wrong, they have typographical mistakes.

Comment (AnderBiguri) [reply Python/tigre/algorithms/art_family_algorithms.py]: completetion-> completion

Comment (Ayushsunny) [reply Python/tigre/algorithms/art_family_algorithms.py]: Sir, I have amend the changes as said by you.

**Review by AnderBiguri — CHANGES_REQUESTED** [review]
This code does not run as it has a simple yet mayor python error. Please do not submit untested PRs to TIGRE.

**Comment by Ayushsunny** [comment]
> This code does not run as it has a simple yet mayor python error. Please do not submit untested PRs to TIGRE.

I've tested it and had attached screenshot too if you rem? in last PR?

**Comment by AnderBiguri** [comment]
That screenshot must be from some other code, as I have literally tested this and has an obvious problem that is intrinsic to python basics. Please test it again to see it.

**Comment by Ayushsunny** [comment]
> That screenshot must be from some other code, as I have literally tested this and has an obvious problem that is intrinsic to python basics. Please test it again to see it.

No the screenshot was from this code only.
So what error you're getting ?

**Comment by AnderBiguri** [comment]
Please, test the code to see it yourself, it takes you a second to do so. 

This is already taking me x5 times the time it would take me to add this code myself, be respectful of that please.

**Comment by Ayushsunny** [comment]
> Please, test the code to see it yourself, it takes you a second to do so.
> 
> This is already taking me x5 times the time it would take me to add this code myself, be respectful of that please.

okay sir, do it yourself or assign someone else.
I cannot work on this org anymore.
Thank you for your help.

**Comment by Ayushsunny** [comment]
> Please, test the code to see it yourself, it takes you a second to do so.
> 
> This is already taking me x5 times the time it would take me to add this code myself, be respectful of that please.

okay sir, do it yourself or assign someone else.
I cannot work on this org anymore.
Thank you for your help.

**Pull request closed without merging**